src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
Not necessarily, I was just starting to upgrade, and this kept me from having to do all at once.
I bet we can remove it at some point.
There are still 5-6 failing tests we should take care of first, before optimizing :)